niyodusengaclement/taskforce-todo-backend
# ch():setup project tests

#### Description of changes made
---

> - Add tests of all endpoints and other useful utils
> - Add Travis CI and Travis CI markdown badge

#### Type of change
---

> **Please select the relevant options**



#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/niyodusengaclement/taskforce-todo-backend.git), and `cd [the project name]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm install` to install the project dependencies
- Run `npm test` to run tests locally
- .....

#### Checklist:



#### Screenshots (if appropriate)
